src/librustc_resolve/diagnostics.rs

```diff
@@ -916,19 +916,21 @@ An import was unresolved. Erroneous code example:
 use something::Foo; // error: unresolved import `something::Foo`.
 ```
 
-Please verify you didn't misspell the import name or the import does exist
-in the module from where you tried to import it. Example:
+Paths in `use` statements are relative to the crate root.  To import items
+relative to the current and parent modules, use the `self::` and `super::`
+prefixes, respectively. Also verify you didn't misspell the import name or the
```

Comment:
"Also verify you didn't misspell the import name or the import does exist" -> "Also verify you didn't misspell the import name or that it does exist"